samples/react-native-macos/src/components/UserFeedbackModal.tsx

```diff
@@ -65,6 +65,36 @@ export function UserFeedbackModal(props: { onDismiss: () => void }) {
             }}
           />
           <View style={styles.buttonSpacer} />
+          <Button
+            title="Send feedback with attachment and tags"
+            color="#6C5FC7"
+            onPress={async () => {
+              onDismiss();
+
+              const base64Image =
+                'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mP8/wcAAgUBA+kZYq8AAAAASUVORK5CYII=';
```

Comment:
For simplicity avoiding extra lib is okay. I'm not sure if this will get correctly recognized in Sentry.

The data type is `UInt8Array | string`, but when string is used it's expected the string is the raw data.